### DISCUSSION TITLE
Are the generated WKWebView MacOS projects compatible with iOS?

### DISCUSSION THREAD

**Discussion by RagnowProductions** [TurboWarp/packager#784] [Q&A] [answered]
I have a project that has been packaged as a MacOS app, and I am wondering if I can

1. Upload it to the App Store
2. Download and play it on my iPad

Is this possible?

**Comment by GarboMuffin** [comment, marked as answer]
No

**Reply by RagnowProductions** [reply]
Ok good to know

is there anything that is compatible as an iOS app?
